src/_stories/mockdata/mock-selected-entities.js

```diff
@@ -0,0 +1,2360 @@
+export const mockSelectedEntities = [
```

Comment:
I'm not a fan of checking in literal mock data, especially in this quantity. It tends to make searching the code base harder. Can we use something like `faker.js` to generate mock data on the fly instead?

Reply:
I removed the mock data. For the purposes of this PR I updated my function to generate the mock data needed to test the behavior of the panel (select all/none, enable/disable of the submit button, scrolling, etc...) If you'd prefer that I pull in the library now, let me know. Otherwise I will take a look at faker.js later and see how it will integrate. Thanks!